### DISCUSSION TITLE
Como ir para o próximo nível

### DISCUSSION THREAD

**Discussion by igorlvicente** [backend-br/forum#130] [Q&A]
Há muitas discussões em diversas comunidades sobre como começar na área, mas me parece que há bem poucos sobre como melhorar quando você está num nível médio para avançado.

Peço-vos que postem dicas, sugestões, apontamentos, etc.

Começo com uma pergunta: Mestrado é uma boa ou MBA é o suficiente?
Muitos dizem que a faculdade não é necessária na nossa área, mas minha experiência pessoal me diz o contrário. A faculdade, embora não tenha me preparado diretamente pro mercado, preparou-me para uma vida voltada aos estudos/ciência, o que indiretamente me tornou um profissional melhor do que eu poderia ter virado apenas com cursos de programação.
Pra quem fez mestrado ou doutorado, vocês pensam parecido em relação a essa pós graduação ou realmente acham que não vale a pena fazer?

**Comment by jeancsanchez** [comment]
Eu concordo em partes com você @igorlvicente . 

IMHO: Acho que criou-se um mito na nossa área onde é dito para as pessoas que faculdade não é relevante, seja pra iniciar ou para se especializar. Por experiencia própria, eu só consegui participar de um processo seletivo na época em que iniciei (~2015) quando eu me matriculei em um curso de ensino superior na área de TI.

Os chamados "autodidatas" são pessoas de um perfil muito específico que conseguem aprender e aplicar na pratica o que aprendeu sem precisar de muita ajuda, e não é todo mundo que tem esse perfil. A grande maioria das pessoas vão precisar de uma faculdade para guiá-las (sem contar no networking e oportunidades que surgem no meio acadêmico).

Ou seja, pra você se destacar no mercado sem um diploma, você precisa ser **MUITO BOM** e provar isso na prática! Com portfolio, projetos pessoais funcionando em produção, etc. 

Por outro lado, só ter um diploma pendurado na parede também não prova nada! Você vai precisar mostrar seu valor na prática também, mas pelo menos você terá mais chances e o voto de confiança dos empregadores provando que no mínimo passou "X" anos estudando sobre um assunto.

Particularmente, eu decidi fazer uma graduação e uma pós em engenharia de software mais como "garantia" mesmo, mas eu vejo que na prática não influencia em nada em questão de cargo e salário pelas empresas por onde passei. O que importa no final é como você aplica esse conhecimento. Tanto é verdade que eu trabalhava ao lado de dois mestrandos e ganhávamos o mesmo salário antes de eu fazer a pós.

Pra finalizar, o que eu venho percebendo durante minha carreira é que os profissionais que têm se valorizado/destacado são os profissionais que dominam **AS BASES**! Principalmente os peseudo-autodidatas têm ignorado coisas básicas da computação que hoje em dia são pouco cobrados nas entrevistas de emprego devido a alta demanda por mão de obra, mas a longo prazo quando a bolha dev estourar de vez, os que aprenderam de verdade (seja sozinhos ou na faculdade) serão os únicos sobreviventes.

Resumindo: aprender sozinho é para poucos, faculdade abre portas, mas no final o que importa é mostrar na prática.
